### ISSUE TITLE
SolidStart barebones example is not building

### ISSUE THREAD

**Issue by JaielZeus** [solidjs/solid-start#913]
I cannot build a barebones starter template.

To replicate it simply follow the instructions on Solidstart:

```
> npm init solid@latest
...
<choosing bare>
<opting out of SSR>
<enabling typescript>
...
> npm install
> npm run build
```
Nothing happens except a ``.solid`` folder is created in the root but it is empty. On top of that the process of "building" is indefinitely stuck. I let it run for an hour while watching a Game Of Thrones episode but it did not finish it's life-long mission of building this starter app unfortunately in that short amount of time. See the screenshot what I mean.

![image](https://github.com/solidjs/solid-start/assets/42944799/4d445bd0-f063-412b-89fc-1cba9d677d62)

**Comment by caseybaggz** [comment]
@JaielZeus I noticed this happened to me when I got an optional question of:

> "Directory not empty. Continue?" ... yes

Which I think might be the issue because once I killed the directory and re-ran it multiple times with different options of the "bare" example it worked every time. Both with NPM and Pnpm.

Would you mind trying to run it again? It might also help to know your environment info (see example below):

```bash
  System:
    OS: Linux 5.10 Debian GNU/Linux 9 (stretch)
    CPU: (8) arm64
  Binaries:
    Node: 14.17.0 - /usr/local/bin/node
    Yarn: 1.22.5 - /usr/local/bin/yarn
    npm: 6.14.13 - /usr/local/bin/npm
  npmPackages:
   solid-start: 1.0.0 => 1.0.0
render: shell
```

**Comment by NathanSkilton** [comment]
I found the issue happens when the project path contains spaces. I'm on Windows.

**Comment by atk** [comment]
Have you tried WSL?

**Comment by JaielZeus** [comment]
> @JaielZeus I noticed this happened to me when I got an optional question of:
> 
> > "Directory not empty. Continue?" ... yes
> 
> Which I think might be the issue because once I killed the directory and re-ran it multiple times with different options of the "bare" example it worked every time. Both with NPM and Pnpm.
> 
> Would you mind trying to run it again? It might also help to know your environment info (see example below):
> 
> ```shell
>   System:
>     OS: Linux 5.10 Debian GNU/Linux 9 (stretch)
>     CPU: (8) arm64
>   Binaries:
>     Node: 14.17.0 - /usr/local/bin/node
>     Yarn: 1.22.5 - /usr/local/bin/yarn
>     npm: 6.14.13 - /usr/local/bin/npm
>   npmPackages:
>    solid-start: 1.0.0 => 1.0.0
> render: shell
> ```

I ran again and it worked just. It was because of @NathanSkilton though. He is totally correct about the issue occuring because of paths with spaces in them:

> I found the issue happens when the project path contains spaces. I'm on Windows.

This is definitely the cause of my building failing, **which only occurs when using non-SSR option**. With SSR and typescript enabled it builds successfully. Without it it does not!

Thank you @NathanSkilton you found a really strange behaviour

**Comment by edivados** [comment]
Path to vite config is unquoted.
https://github.com/solidjs/solid-start/blob/8cfe543c18c2ee2aed30478da9893c37ec8be7f7/packages/start/bin.cjs#L370-L378

Changed it to the following for a quick test and works.
```
...(config ? ["--config", `"${config.configFile}"`] : []),
```

**Comment by edivados** [comment]
Fixed in `0.2.27`
